### PR TITLE
Telemetry Article

### PR DESCRIPTION
**Pull request by anodium**
Article explaining telemetry, following is known to be missing and should be fixed before merge:

- [ ] Screenshots for opt-out flow (need to make sure *everyone* knows exactly what to do so they don't come around and complain)
- [ ] Article banner (which @bunnei has said they'll handle
- [ ] Forum post ID
- [ ] That link I mention on the last sentence, which *should* point to an article that quoted Microsoft engineers fucking up Windows 8 so badly because their analytics grossly misrepresented their customers. (Gotta get that SEO going 😛)

### CODE REVIEW

**Comment by anodium** [comment]
Requesting these people for review:

- [ ] @chris062689
- [ ] @jroweboy
- [ ] @Schplee
- [ ] @bunnei
- [ ] @j-selby
- [ ] @Subv

Doing it by mass-ping because apparently I can't request reviews with GitHub's UI if I only have read privileges to the repo.

File: site/content/entry/telemetry-and-why-thats-a-good-thing.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
++++
+date = "2017-08-02T21:02:00-04:00"
+title = "Telemetry (And Why That's A Good Thing)"
+tags = [ "citra-release" ]
+author = "anodium"
+forum = 0000
++++
+
+Citra has some [issues](https://github.com/citra-emu/citra/issues), we've always been transparent in what they are, and have always allowed anyone to contribute to it. Unfortunately though, most contributions are made by [a small minority of developers](https://github.com/citra-emu/citra/graphs/contributors). These developers have found it difficult to prioritize their efforts, since the issues are mostly reported outside of the GitHub tracker, on the forums, on chats, and elsewhere. Not to mention that most of the reports are too vague, are missing vital information such as the logs or system info, or both.
```
Comment (j-selby) [inline site/content/entry/telemetry-and-why-thats-a-good-thing.md]: "we've always been transparent *about* what they are"

Comment (j-selby) [inline site/content/entry/telemetry-and-why-thats-a-good-thing.md]: That first sentence doesn't flow well, though that might just be me.

File: site/content/entry/telemetry-and-why-thats-a-good-thing.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Because of this, the Citra team has put together a framework to report data about our users to our server, and use that data to find things like where emulated games crash in Citra most often, what are the most popular games and hardware configurations, and so on. We had considered including this in last month's progress report, but we decided that it would be best to publish this on its own, so it gets the attention it deserves. We're very aware that privacy is important to many of our users, and so from the earliest planning stage we knew we must be as transparent and open about this as possible.
```
Comment (j-selby) [inline site/content/entry/telemetry-and-why-thats-a-good-thing.md]: "what are the most popular games, hardware configurations, and so on"

File: site/content/entry/telemetry-and-why-thats-a-good-thing.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+Then, head into the Services tab.
+</p>
+
+<p style="text-align: left; padding: 1%">
+<img style="padding: 0% 0% 1% 0%" height="75%" width="75%" alt="IMAGE DESCRIPTION HERE" src="IMAGE LINK HERE" />
+<br />
+And finally, untick the box titled "Report Telemetry Data".
+</p>
+
+Telemetry is an extremely useful developer tool, as they allow the developer to be more aware of the users' needs and priorities, rather than be forced to guess, usually incorrectly. But it is only as useful as the data obtained, which is why we urge users to not opt-out, so the data can be as accurate and correct as possible. [Who knows what could happen if our telemetry data doesn't represent our users accurately...](<!-- Link to that article Microsoft published on how they got Windows 8 so wrong because the analytics didn't represent their users accurately. -->)
```
Comment (j-selby) [inline site/content/entry/telemetry-and-why-thats-a-good-thing.md]: "as they allow the developer to be more aware of users' needs and priorities accurately, rather then guessing."

**Comment by chris062689** [comment]
FYI - with a change to telemetry that I'll be making tonight, we will no longer store any variation of IP address in the database (raw value or hash)

File: site/content/entry/telemetry-and-why-thats-a-good-thing.md (hunk 1 above)
Comment (Schplee) [inline site/content/entry/telemetry-and-why-thats-a-good-thing.md]: I agree that the structure of the first sentence is a bit awkward.

File: site/content/entry/telemetry-and-why-thats-a-good-thing.md (hunk 2 above)
Comment (Schplee) [inline site/content/entry/telemetry-and-why-thats-a-good-thing.md]: "find things like" can be changed to "discover" - "and use that data to discover what the most popular games and hardware configurations are, as well as where emulated games crash most often in Citra, and more."

**Pull request closed without merging**